copilot/rails/addons/firehose-iam.yml

```diff
@@ -0,0 +1,15 @@
+Resources:
+  FirehosePolicy:
+    Type: AWS::IAM::ManagedPolicy
+    Properties:
+      PolicyDocument:
+        Version: 2012-10-17
+        Statement:
+        - Effect: Allow
+          Action:
+          - firehose:PutRecordBatch
+          Resource: "*"
```

Comment:
Must `*` instead of Kinesis Data Firehose Steam name (ex. `chronos-rails-datadog`)

```
Resource chronos-rails-datadog must be in ARN format or "*". (Service: AmazonIdentityManagement; Status Code: 400; Error Code: MalformedPolicyDocument; Request ID: f01bf0c3-13e7-48a4-8997-18b4f417103f; Proxy: null)
```